.drone.star

```diff
@@ -152,3 +201,15 @@ def append_depends_on(step, refs):
 
     step["depends_on"] = deps
     return step
+
+def get_commit_name(arch=""):
+    if arch:
+        return "git-${DRONE_COMMIT_SHA:0:7}-" + arch
```

Comment:
nit: with starlark you can pass `ctx` to this and [use those values](https://docs.drone.io/pipeline/scripting/starlark/#context-variable) instead of relying on variable substitution:

```python
return "git-{}-{}".format(ctx.build.commit[:7], arch)
```

Reply:
Ah that's neat. I ended up not using the ctx variables because the resulting .drone.yml from `drone starlark convert --format` would be `git--arm64`, making it hard to verify the result.

Reply:
You can pass a dummy value, which allows you to verify the output even a little more clearly than variable substitution:
```
drone starlark --build.commit 1234567890 --format --stdout
```